tools/test/h5copy/CMakeTests.cmake

```diff
@@ -38,16 +46,49 @@
       h5copy_help2.ddl
   )
 
+  # Testfile directory setup
   file (MAKE_DIRECTORY "${PROJECT_BINARY_DIR}/testfiles")
 
-   
+  # Generate testfiles for VOL connector(s) through script, if enabled
+  if (HDF5_BUILD_GENERATORS)
```

Comment:
Since the `HDF5_BUILD_GENERATORS` option has been removed this clause can be removed